frontend/javascripts/dashboard/dataset/queries.tsx

```diff
@@ -496,11 +496,6 @@ function diffDatasets(
     .map((id) => newDatasetsDict[id])
     .filter((newDataset) => {
       const oldDataset = oldDatasetsDict[newDataset.id];
-      // Ignore updates when migrating to new renamable datasets.
-      const didUpgradeToRenamableDS = ("directoryName" in oldDataset && "directoryName" in newDataset); // TODO: Can be remove after a few weeks / months.
-      if(didUpgradeToRenamableDS) {
-        return false;
-      }
```

Comment:
Or we remove whole check as we think that most of the user already refreshed their cached datasets after the dataset renaming pr. That's why this TODO comment was here.